src/main/java/vectorwing/farmersdelight/integration/jei/FDRecipes.java

```diff
@@ -1,5 +1,7 @@
 package vectorwing.farmersdelight.integration.jei;
 
+// TODO: Uncomment this once JEI updates.
```

Comment:
Due to how long I took to review this, both JEI and CT have already ported to both 1.20.2 and 1.20.4... Sorry about that. x)